### DISCUSSION TITLE
Issue opening .crc files from Oxford instrument ( MTEX 5.9)

### DISCUSSION THREAD

**Discussion by Belt91** [mtex-toolbox/mtex#1841] [Ask Anything]
Hi,

I am struggling to open .crc files with MTex. Once I used to do it by using the EBSD.load('filepath\filename.crc') function or by using the import wizard. Now, in both cases, it raises an error saying that MTex could not detect the file format ( see picture below)

![Error_crc_Mtex](https://github.com/mtex-toolbox/mtex/assets/134724391/7f5df95e-b2df-4983-9d32-31b1d42d7538)

If I try to specify the interface like EBSD.load('filepath\filename.crc','interface, 'crc' ) 
it even tells me that the file format does not match the data...

![image](https://github.com/mtex-toolbox/mtex/assets/134724391/b22847ae-256f-4b89-810d-872ca6063c0e)

I really appreciate any help you can provide.

Marco

**Comment by ralfHielscher** [comment]
Could you somehow provide the data?

Ralf.

**Reply by Belt91** [reply]
Hi Ralf,

thanks for your reply. Apparently, the problem is in the data. I retrieved the old data I used to practice with and it goes fine. The instrument is the very same ( A Zeiss EVO)  and the software is as well (HKL flamenco). I will try to figure out what happens...in the meantime, if you have experience with some similar issues, please share :D

**Reply by Belt91** [reply]
> Could you somehow provide the data?
> 
> Ralf.

Hey Ralph, I asked my boss and he agrees in sharing one example file. I send it to you in the e-mail you used during the MTEX workshop.

Best,

Marco

**Comment by lfgmorales** [comment]
Hi

The *.crc & *.cpr pair are proprietary files and change considerably from Aztec to Aztec version, so is it always better to export your data as *.ctf, which is a text file and normally does not give problems

Best

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 26 May 2023, at 19:37, Belt91 ***@***.***> wrote:

﻿

Could you somehow provide the data?

Ralf.

Hey Ralph, I asked my boss and he agrees in sharing one example file. I send it to you in the e-mail you used during the MTEX workshop.

Best,

Marco

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1841#discussioncomment-6014302>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN7EMRL7AO77WIGWBD3XIDS6DANCNFSM6AAAAAAYQN26OQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1841#discussioncomment-6014302", "url": "https://github.com/mtex-toolbox/mtex/discussions/1841#discussioncomment-6014302", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by Belt91** [comment]
Dear Luiz,

Thank you for the advice, I Will keep It at mind!

Best,

Marco

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Xavi007** [comment]
The error message you shared suggests that MTex is unable to detect the file format of the .crc file. This could be due to various reasons, such as incorrect file path, file corruption, or compatibility issues. Here are a few steps you can take to troubleshoot the problem:

Verify the file path: Double-check that the file path you are providing is correct and that the file exists in that location. Make sure there are no typos or missing characters in the file path.

Check file compatibility: Ensure that the .crc file you are trying to open is compatible with the version of MTex you are using. Sometimes, file formats can change between different software versions, leading to compatibility issues. Refer to the MTex documentation or user guide to verify the supported file formats for your version.

File integrity: Confirm that the .crc file is not corrupted. You can try opening the file with a different software or text editor to see if it opens without any errors. If the file is indeed corrupted, you might need to obtain a fresh copy from the original source.

Seek help from the MTex community: If the issue persists, it would be helpful to reach out to the MTex community for further assistance. The MTex website or forum may have dedicated support channels or a community of users who can provide insights or solutions to specific issues.
